buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -458,10 +460,14 @@ class BeamModulePlugin implements Plugin<Project> {
 
     project.ext.mavenGroupId = 'org.apache.beam'
 
-    // Automatically use the official release version if we are performing a release
-    // otherwise append '-SNAPSHOT'
-    project.version = '2.51.0'
```

Comment:
these changes / removing this line possibly caused internal import failure @damccorm

Was this intended / or can we add it back?

Reply:
This also caused the version number of some test became `2.51.0-SNAPSHOT-SNAPSHOT`  e.g. validate runner tests

Reply:
Ah, this is probably also what is impacting the nightly snapshot then.

Reply:
I'm not sure that we actually need it for the snapshots to work